index.js

```diff
@@ -1,58 +1,108 @@
 // implement your API here
 const express = require("express");
 const server = express();
-const db = require("./data/db.js");
 const port = 8000;
+const db = require("./data/db.js");
+const cors = require("cors");
+
+server.use(express.json());
+server.use(cors());
+
+const sendError = (res, status, msg) => {
```

Comment:
so this is pretty cool, though I would say it is kind of a waste of time as you'll end up writing about the same in the end (probably more). The style points don't outweigh just being able to the flexibility of being able to put extra data you might need. Such as the fact that in error messages you like in line 35 you're not properly catching the error so if there is one it will be looked at as an unresolved promise which will simply break your server.